internals/secrethub/signup.go

```diff
@@ -164,5 +164,6 @@ func (cmd *SignUpCommand) Run() error {
 	}
 
 	fmt.Fprintln(cmd.io.Stdout(), "Signup complete! You're now on SecretHub.")
+	fmt.Fprintf(cmd.io.Stdout(), "We've send an email to %s Please verify your email address to continue.\n", cmd.email)
```

Comment:
`Please verify your email address to continue, we've sent a verification mail to %s.\n"`

Reply:
Yes, that's better :+1: